docs/model-dev-guide/api-guides/apis-howto/api-core-ug.rst

```diff
@@ -334,6 +334,7 @@ settings in our experiment configuration file:
 -  ``max_trials``: 500 (This is the maximum number of trials the searcher should run.)
 -  ``max_length``: 20 epochs (The max length of a trial. For more information, visit Adaptive ASHA
```

Comment:
-  ``max_length``: 20 epochs (The max length of a trial.  This is a required parameter for the SearcherContext to function properly.)